db/schema.rb

```diff
@@ -33,7 +33,6 @@
   end
 
   create_table "users", force: :cascade do |t|
-    t.string "uid"
     t.string "name"
     t.string "email"
     t.datetime "created_at", null: false
```

Comment:
I think we want this :uid right? When I ran migrations it was deleted. Wondering if we maybe need to add that in as a migration too?

Reply:
@twarbelow thoughts?